blueprint/api.apib

```diff
@@ -2174,6 +2217,14 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Slugged Route Organization Resource)
 + include JSON API Version
 
+## StripeCard (object)
++ data(StripeCard Resource)
++ include JSON API Version
+
+##StripeCards (object)
++ data(StripeCards Resource)
```

Comment:
I think you need a `StripeCard Response` and a `StripeCards Response`. Look at `Categories` as an example here.